core/backend/src/GeoCoordServices.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelJsNative } from "@bentley/imodeljs-native";
+import { Range2dProps } from "@itwin/core-geometry";
+
+/** Class containing services for working with geographic coordinates.
+ * @public
+ */
+export abstract class GeoCoordServices {
+  /** Get a list of Geographic Coordinate Systems.
+   * @param ignoreLegacy If true, only return GCS's that are not considered legacy.
```

Comment:
do we have any docs on what is defined as "legacy" gcs?

Reply:
In the iTwinjs-core we instead use the term 'deprecated' and is represented as a bool in the HorizontalCRS props at: https://github.com/iTwin/itwinjs-core/blob/9ec9aab98c6893f49065e6a7e14862115eb8f650/core/common/src/geometry/CoordinateReferenceSystem.ts#L97
Internally in the CSMAP compliant database we mark deprecated CRS by indicating they belong to the LEGACY group. We could normalize to deprecated and rename legacy everywhere.
The meaning of 'deprecated' is the same as in the official EPSG database. The entry is still valid and can be used when older data is being referenced, but shall not be used for newer dataset.

Reply:
So probably the option should be `includeDeprecated` (default false) instead of `ignoreLegacy`.

Reply:
The "includeDeprecated" parameter is not necessary anymore. We decided to return a "deprecated" field with the results.